packages/synchronize/lib/src/journal_entry.dart

```diff
@@ -0,0 +1,44 @@
+part of '../synchronize.dart';
+
+/// Stores a single entry in the [SyncJournal].
+///
+/// It contains an [id] and ETags for each object, [etagA] and [etagB] respectively.
+@immutable
+@JsonSerializable()
```

Comment:
This is the reason I don't like the part stuff.

I was just searching for the generated file and forgot that it's all in one.